Gemfile.lock

```diff
@@ -528,6 +528,11 @@ GEM
     google-protobuf (4.28.3)
       bigdecimal
       rake (>= 13)
+    google-protobuf (4.28.3-java)
+      bigdecimal
+      ffi (~> 1)
+      ffi-compiler (~> 1)
+      rake (>= 13)
```

Comment:
is this `google-protobuf` addition intentional?

Reply:
It isn't - thank you for the catch